src/components/Forms/StyledComponents/styledEditButton.jsx

```diff
@@ -1,6 +1,9 @@
 import styled from 'styled-components';
 
 const StyledEditButton = styled.button`
+	position: absolute;
+	top: 5px;
+	right: 5px;
```

Comment:
If you change the styling of the parent `<li> `like I mentioned in my comment, you can remove these lines 🧹 for the `StyledEditButton.js `AND for the `StyledSpan.js` **(which is not part of this PR)**  .